src/uct/ib/mlx5/rc/rc_mlx5_iface.c

```diff
@@ -930,6 +930,9 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_t,
         return status;
     }
 
+    uct_rc_mlx5_ddp_init_if_available(&self->super, md,
```

Comment:
shall we also add to gga INIT_FUNC(*iface?

Reply:
or rather merge this function inside `uct_rc_mlx5_dp_ordering_ooo_init()`?

Reply:
It is only available for rc and dc so I'm not 100% sure about this, @yosefe ?

Reply:
ddp is not relevant for gga transport